examples/grommet/src/widgets/templates/Field.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box } from 'grommet';
+import { PropsWithChildren, useMemo } from 'react';
+
+export interface FieldProps {
+  error?: Boolean;
+  message?: string;
+  fieldClasses?: string[];
+}
+
+export default function Field({
+  fieldClasses = [],
+  error,
+  message,
+  children,
+}: PropsWithChildren<FieldProps>) {
+  const classList = useMemo(() => {
+    const classes = ['form-field', ...fieldClasses];
+    if (error) {
+      classes.push('form-field-error');
+    }
+    return classes.join(' ');
+  }, [fieldClasses, error]);
+
+  return (
+    <Box className={classList} pad="medium" style={{ flexGrow: '1' }}>
+      {children}
+      {message && <span className="form-field-message">{message}</span>}
+    </Box>
+  );
+}
```

Comment:
I notice you don't use the component FormField of grommet, I think to be really a grommet example, this should be used https://v2.grommet.io/formfield
<FormField label="Field label">
  <TextInput placeholder="type here" />
</FormField>